de.metas.contracts/src/main/java/de/metas/contracts/commission/commissioninstance/services/CommissionTriggerRequest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * #%L
+ * de.metas.contracts
+ * %%
+ * Copyright (C) 2019 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.contracts.commission.commissioninstance.services;
+
+import de.metas.contracts.commission.commissioninstance.businesslogic.CommissionPoints;
+import de.metas.invoicecandidate.InvoiceCandidateId;
+import de.metas.util.lang.Percent;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NonNull;
+
+import java.time.Instant;
+
+@Data
```

Comment:
since you don't use the setters, i suggest that you use lombok's `@Value` instead
(*if* it made sense to use `@Data`, then you should make the classe's fields `private`. but with  `@Value`, that's already done for you)